csunplugged/tests/topics/urls/test_glossary_url.py

```diff
@@ -0,0 +1,13 @@
+from tests.BaseTestWithDB import BaseTestWithDB
+from django.urls import reverse
+
+
+class GlossaryURLTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.language = 'en'
+
+    def test_valid_glossary_url(self):
+        url = reverse('topics:glossary')
+        self.assertEqual(url, '/en/topics/glossary/')
```

Comment:
No, as the response is calculated by a view so it's in the view tests. The URL tests only check the URLs calculate correctly.

I did notice the existing tests were checking status in both the URL and view tests, but it made sense for it only to be tested in the views.